Added welcome page and fixes minor tutorial issues (#340)
IrisShaders/DocsPage: src/content/docs/reference/overview.mdx

---

> title: Welcome

Reviewer: ```suggestion
title: Welcome...
```

import { LinkCard, CardGrid } from '@astrojs/starlight/components';

> To the Iris docs!

Reviewer: ```suggestion
...to the Iris docs!
```

> Iris is a shader loader that is integrated with sodium, with support for modern shaders, built from the ground up for ease of use. Weather your new to shader development, want to migrate your shader from optifine, or just need a refresher on what Iris can do, you have come to the right place. 

Reviewer: ```suggestion
Iris is a shader loader that is integrated with [Sodium](https://modrinth.com/mod/sodium/), with support for modern shaders, built from the ground up for ease of use. Whether you're new to shader development, want to migrate your shader from Optifine, or just need a refresher on what Iris can do, you can find everything you need to know right here.
```

Reviewer: Also I feel like "modern shaders" is vague and doesn't mean much.

Author: Sounds good tho

> If you can't find what you are looking for, reach out to us on the [Iris Discord](https://discord.com/invite/jQJnav2jPu).
> 
> Now, if you are feeling a bit lost, why dont you look into the items bellow.

Reviewer: ```suggestion
If you can't find what you are looking for, reach out to us on the [Iris Discord](https://discord.com/invite/jQJnav2jPu).

If you're feeling a bit lost, why don't you have a look at the items below.
```
